extensions/positron-r/src/test/mocha-setup.ts

```diff
@@ -9,7 +9,10 @@ import * as testKit from './kit';
 export let currentTestName: string | undefined;
 
 suiteSetup(async () => {
-	// Set Ark log level to TRACE for easier debugging of tests
+	// Set global Positron log level to trace for easier debugging
+	await vscode.commands.executeCommand('_extensionTests.setLogLevel', 'trace');
```

Comment:
Nice little command that's only valid when extension tests are running: https://github.com/posit-dev/positron/blob/d46ce10821cbc68129713f2e0596363e25fb6222/src/vs/workbench/api/browser/mainThreadLogService.ts#L78

It allows modifying the global vscode log level.